### DISCUSSION TITLE
Usage on macOS

### DISCUSSION THREAD

**Discussion by astamm** [vanderschaarlab/synthcity#232] [General]
Hi,
I just listened to Dr van der Schaar keynote talk at the RSS conference and installed the soft on my Mac M1 laptop. However, starting with

```
from synthcity.plugins import Plugins
```

I get straightaway the fatal error that `omp.h` file is not found which, I believe, is linked to mac-packaged clang does not support openMP anymore.

Any clue on how I can use your software on macOS ?

Thanks!

**Comment by ShengzheXu** [comment]
`conda install py-xgboost` this works for me.

[1] https://stackoverflow.com/questions/61971851/getting-this-simple-problem-while-importing-xgboost-on-jupyter-notebook
